cabal.project

```diff
@@ -161,8 +161,8 @@ constraints:
   , Cabal >= 3.4.0.0
   , async-timer >= 0.2.0.0
   , unliftio-core >= 0.2.0.1
-  , cardano-api >= 1.35.4
-  , cardano-node >= 1.35.4
+  , cardano-api >= 1.35.6
+  , cardano-node >= 1.35.6
```

Comment:
These bounds are just lower bounds.

What we actually get is:
```
$ cardano-node --version
cardano-node 1.36.0 - linux-x86_64 - ghc-8.10
git rev 84a871ed86fc1b86018231b9cb24ba7b83493c0d
```

We could just set this to:
```
  , cardano-api >= 1.36.0
  , cardano-node >= 1.36.0
```
Or
```
  , cardano-api  == 1.36.0
  , cardano-node == 1.36.0
```